src/rewrite-imports.ts

```diff
@@ -61,17 +62,16 @@ async function transformHtmlImports(code: string, replaceImport: (specifier: str
 }
 
 export async function transformFileImports(
-  code: string,
-  fileName: string,
+  {baseExt, code}: SnowpackFile,
   replaceImport: (specifier: string) => string,
 ) {
-  if (fileName.endsWith('.js')) {
+  if (baseExt === '.js') {
```

Comment:
With `.baseExt` built-in, we don’t have to do as much work to tell each time what we’re dealing with. We’ll just trust that the file interface has figured it out already.